bitbots_vision/config/visionparams.yaml

```diff
@@ -1,29 +1,49 @@
-neural_network_model_path: '2019_07_03_jonas_yolo'
-ball_fcnn_publish_debug_img: false  # toggles publishing of the fcnn heatmap image for debug purposes
+vision_parallelize: true
+vision_blind_threshold: 30
+
+ROS_img_msg_topic: 'image_raw'
+ROS_img_msg_queue_size: 1
+ROS_fcnn_img_msg_topic: 'fcnn_image'
+ROS_field_boundary_msg_topic: 'field_boundary_in_image'
+ROS_ball_msg_topic: 'ball_in_image'
+ROS_goal_msg_topic: 'goal_in_image'
+ROS_obstacle_msg_topic: 'obstacles_in_image'
+ROS_line_msg_topic: 'line_in_image'
+ROS_dynamic_color_space_msg_topic: 'dynamic_color_space'
+ROS_debug_image_msg_topic: 'debug_image'
+ROS_debug_fcnn_image_msg_topic: 'fcnn_debug_image'
+ROS_field_mask_image_msg_topic: 'field_mask'
+ROS_dynamic_color_space_field_mask_image_msg_topic: 'dynamic_field_mask'
+
+fcnn_model_path: '2019_06_timon_basler'
+yolo_model_path: '2019_07_03_jonas_yolo'
 ball_fcnn_threshold: .6  # minimal value for a candidate to be considered
 ball_fcnn_expand_stepsize: 4
 ball_fcnn_pointcloud_stepsize: 10
-ball_fcnn_shuffle_candidate_list: false  # shuffles the list of possible candidate points
 ball_fcnn_min_ball_diameter: 15
 ball_fcnn_max_ball_diameter: 150
 ball_fcnn_publish_output: false
 ball_fcnn_publish_field_boundary_offset: 5
 
-field_color_detector_path: 'sydney_2_7_2019_interpolated.txt'
+ball_detector_method: 'yolo_darknet'  # fcnn or yolo_opencv or yolo_darknet or dummy
+ball_candidate_field_boundary_y_offset: 0
+ball_candidate_rating_threshold: 0.5
+
+vision_use_sim_color: false
+field_color_detector_path: 'f031_basler.txt'
 field_color_detector_path_sim: 'simColor_interpolated.yaml'
-field_color_detector_use_hsv_green: false
```

Comment:
this is currently not implemented, but I think we should do it for demos and stuff. In some other issue/PR of course.

Reply:
Same as with the line blur. Is it okay if we open an issue (maybe post version 1) and remove it from the current vision config.

Reply:
After the issue is opened, I'm fine with it, too.